index.ts

```diff
@@ -605,15 +624,21 @@ async function copyIntoSnowflake({ cache, global, jobs }: Meta<SnowflakePluginIn
     await cache.set('lastRun', timeNow)
     console.log(`Copying ${String(filesStagedForCopy)} from object storage into Snowflake`)
     try {
-        await global.snowflake.copyIntoTableFromStage(filesStagedForCopy, global.purgeEventsFromStage, global.forceCopy, global.debug)
+        await global.snowflake.copyIntoTableFromStage(
+            filesStagedForCopy,
+            global.purgeEventsFromStage,
+            global.forceCopy,
+            global.debug
+        )
     } catch {
         await jobs
             .retryCopyIntoSnowflake({ retriesPerformedSoFar: 0, filesStagedForCopy: filesStagedForCopy })
             .runIn(3, 'seconds')
-        console.error(`Failed to copy ${String(filesStagedForCopy)} from object storage into Snowflake. Retrying in 3s.`)
+        console.error(
+            `Failed to copy ${String(filesStagedForCopy)} from object storage into Snowflake. Retrying in 3s.`
+        )
     }
-    await cache.expire(REDIS_FILES_LIST_KEY, 0)
+    await storage.set(FILES_STAGED_KEY, undefined)
```

Comment:
doing this is the same as `storage.del` - we might as well be explicit.

Reply:
I didn't see it locally but looks like it was because of the plugin scaffolding version - updated.